diff --git a/helm/kube-vip-cloud-provider/templates/_helpers.tpl b/helm/kube-vip-cloud-provider/templates/_helpers.tpl
index 7365d7d..3d02291 100644
--- a/helm/kube-vip-cloud-provider/templates/_helpers.tpl
+++ b/helm/kube-vip-cloud-provider/templates/_helpers.tpl
@@ -41,6 +41,7 @@ helm.sh/chart: {{ include "kube-vip-cloud-provider.chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 {{- end }}
 
 {{/*
